lib/mapper.js

```diff
@@ -112,14 +113,30 @@ function common(facade){
   if (facade.city()) ret.city = facade.city();
   if (facade.region()) ret.region = facade.region();
 
-  if (os && os.toLowerCase() == 'ios') {
+  function mobileDeviceType(){
```

Comment:
This should be declared outside of this function; it's being reconstructed and GCed every time this function is invoked